README.md

```diff
@@ -52,109 +55,87 @@ As an Addon reviewer you will:
+Addons versions are submitted by submitting a pull request, adding a file for that version of the addon.
 
 ### Layout
 
 Root directory of repository:
  - `readme.md` - A guide for submission
- - `addons/owner1/repo1.commits`
- - `addons/owner1/repo2.commits`
- - `addons/owner2/repo3.commits`
+ - `addons/publisher1/addon1/majorVersion.minorVersion.patch.json`
+ - `addons/publisher1/addon2/majorVersion.minorVersion.patch.json`
+ - `addons/publisher2/addon3/majorVersion.minorVersion.patch.json`
+
+Note: `publisher.addonName` will become the addon ID, and must be unique and match the addon ID from the addon manifest.
 
-Example for the NV Access addon, 'NVDA - OCR': `addons/nvaccess/nvda-ocr.commits`
+Example for the NV Access addon, 'NVDA - OCR': `addons/nvaccess/nvda-ocr/1.6.0.json`
 
-Contents of a `*.commits` file, is a (newline separated) list of git SHAs, one for each commit which could be available in the store.
-This is intentionally a very simple format to maximise the ease of editing, and minimise the risk of format corruption.
-Commits are used rather than tags because tags can be changed after creation, a SHA collision should be quite difficult to achieve.
-This is important to stop the possibility of a malicious author changing an addon post review, bypassing the review process.
+### Metadata format
+For a full description of the schema see:` _tools/addonVersion_schema.json`
+- This includes an example of the file contents.
 
 ### Submitting an Addon version
 
-Pre-requisites:
-- An addon must be stored on GitHub in a public repository
-- The commit which will be submitted must be a valid NVDA `*.nvda-addon` file if zipped and renamed.
-
-Process:
-1. A PR is created on the `NVDA-Addon-submission` repository
-   - It adds/removes the SHA of the commit (for the submitted version) to the appropriate `addons/owner/repo.commits` file.
-   - Refer to 'Creating the submission PR' below
-1. A bot adds a comment in the PR with links to the addon versions that are added or removed
-   - NOTE: the URL for this link can be is built from 'owner', 'repo', 'commit' of the lines added / removed. 
-   - This allows the reviewer to browse the source online or download the addon with, see (in suffix) heading "Example of bot message for `NVDA-Addon-submission` PR" for an example message.
-1. A review is performed (resulting in: request changes, merge, close)
-   - A bot may perform some automated checks or provide helpful information, refer to 'automated review ideas' below
+#### Pre-requisites:
+- Familiarity with GitHub
+- Familiarity with Git, including working with branches.
+
+Process to add a new NVDA-addon version:
+1. Fork the `addon-store-submission` repository
+1. On a new branch, copy the `_template_addon_release.json` file. 
```

Comment:
```suggestion
2. On a new branch, copy the `_template_addon_release.json` file. 
```

Reply:
This renders to a numbered list in markdown, but stops you from having to renumber (or potentially make mistakes) during editing. Please check the rich diff, or rendered output.

Reply:
Ah great! Thanks, I didn't know this.